internal/trace/context.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright The ORAS Authors.
```

Comment:
Is it OK to copy it directly from ORAS and keep the license header?

Reply:
Yes. We should keep the License also mentions where does we copy the code.